frontend/src/Pages/SignUp/components/InputTextForm.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+export function InputTextForm({ text }) {
```

Comment:
Nice work abstracting this out!  Is this something that is specific to the "SignUp" flow, or is this a component that we can be reusing across other pages as well? (e.g. the calorie calculator).

Reply:
It is a component that could be reused in other pages. Looks like I forgot to change it's directory